ui/appui-react/src/appui-react/statusfields/toolassistance/ToolAssistanceField.scss

```diff
@@ -16,33 +16,14 @@ $key-size-medium: 18px;
 $key-size-large: 36px;
 $key-size-modifier: 24px;
 
-.uifw-statusFields-toolassistance {
-  $font-size-medium: var(--iui-font-size-0);
-  $font-size-small: var(--iui-font-size-0) - 1px;
-
-  overflow: hidden;
-  width: 20em;
-
-  @include for-tablet-landscape-down {
-    width: 16em;
-  }
-
-  @include for-tablet-portrait-down {
-    font-size: $font-size-medium;
-  }
-
-  @include for-phone-only {
-    font-size: $font-size-small;
-    width: 12em;
-  }
-
-  .nz-indicator {
-    display: flex;
-    flex: auto;
-
-    .nz-icons :nth-child(1) {
-      margin-right: 4px;
-    }
+.uifw-statusFields-toolAssistance-toolAssistanceField {
+  width: 30ch;
+  justify-content: flex-start;
+
+  .prompt {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
@GerardasB Jon and I decided to show ellipsis for overflown text here. That means that some prompts will never be fully displayed because they are too long (although they will be fully displayed in the opened popover). Do you think this is fine? We also considered leaving comment somewhere saying that instructions above certain length will not be fully displayed in the `ToolAssistanceField` but I did not find a good place because the `ToolAssistance.createInstrunction` seems to be in `appui-abstract`.

Reply:
Is it only me, or do we need to add some additional spacing to the expand indicator?

Reply:
It still has a capped size. On line 20 the width is set to a static value.

Do you mean additional spacing away from the sides or away from the content of the button?

Reply:
BTW I could increase the spacing by moving the triangle closer to the edge by 1px. My only concern that it might be harder to notice when closer to the edge (although that may not be a bad thing). I cannot increase the spacing in another way because this is a border with absolute positioning. The position closer to the edge is what was used by `ToolAssistanceField` before so maybe we should use that. What do you think?

Current:
![image](https://github.com/iTwin/appui/assets/138560995/12d0a15f-bb04-486a-8ff4-59859b7ef90b)

Moved (and used by `ToolAssistanceField` before:
![image](https://github.com/iTwin/appui/assets/138560995/e0d1c1f7-cd99-431b-a4b2-d46996eec876)

Reply:
I see what you mean, that's why I asked - in case visual change like this is needed in the future +@FlyersPh9

Reply:
@FlyersPh9 Jon, what do you think? Should the indicator be closer to the edge or not?

Reply:
That 1px change is hardly even noticeable.  I think I prefer current over moved example because the bottom of the triangle aligns with the bottom of the dividing lines.